plutus-ledger-api/src/Plutus/V1/Ledger/EvaluationContext.hs

```diff
@@ -31,7 +36,15 @@ mkEvaluationContext newCMP =
 isCostModelParamsWellFormed :: Plutus.CostModelParams -> Bool
 isCostModelParamsWellFormed = isJust . Plutus.applyCostModelParams Plutus.defaultCekCostModel
 
--- only used for testing purposes: make an evaluation context by applying
--- an emptys set of protocol parameters
+-- | The set of valid names that a cost model parameter can take for the specific protocol version.
+-- It is used for the deserialization of `CostModelParams`.
+defaultCostModelParamNames :: Set.Set Text.Text
```

Comment:
I'd drop the "default", there's nothing default about it. It's the *values* of the default cost model which are default.

Reply:
Indeed stupid me